app-modules/knowledge-base/resources/views/filament/pages/badges.blade.php

```diff
@@ -33,7 +33,32 @@
 --}}
 
 <div class="flex gap-2">
+    @if ($getRecord()->status)
+        <x-filament::badge>
+            {{ $getRecord()->status->name }}
+        </x-filament::badge>
+    @endif
     <x-filament::badge>
-        {{ $getRecord()?->status?->name ?? '' }}
+        {{ $getRecord()['public'] ?? false ? 'Public' : 'Internal' }}
     </x-filament::badge>
+    @if (!empty($getRecord()['public']))
```

Comment:
I believe selecting a category is currently optional. And an article needs to be in a category and be public to show up in the portal.
So let's also check here that `$getRecord()->category_id` is also not empty or else the link would not work.

Reply:
condition added. done.